### DISCUSSION TITLE
[Request] Slideshow/Image Widget

### DISCUSSION THREAD

**Discussion by Jacksaur** [glanceapp/glance#259] [General]
Would be great to have a Widget that could cycle through a set of images in a folder over a configurable amount of time for a little extra flair.
This could be possible with just an iFrame to Immich or a similar webapp of course, but it'd be nice to have an integrated solution.

**Comment by svilenmarkov** [comment]
Thanks for the suggestion, I might be able to add something like this in v0.8.0.
